src/pages/rekey-search/rekey-search.selector.ts

```diff
@@ -1,7 +1,19 @@
 import { RekeySearchModel } from './rekey-search.reducer';
+import { isEmpty } from 'lodash';
 
 export const getIsLoading = (rekeySearch: RekeySearchModel) => rekeySearch.isLoading;
 
 export const getHasSearched = (rekeySearch: RekeySearchModel) => rekeySearch.hasSearched;
 
-export const getBookedTestSlot = (rekeySearch: RekeySearchModel) => rekeySearch.bookedTestSlot;
+export const getBookedTestSlot = (rekeySearch: RekeySearchModel) => {
```

Comment:
A ternary operator might simplify this. Potato, potato I guess though.

Reply:
I don't want this to be a trend so writing it this explicitly makes the code **less cool** and hopefully developers will be less encouraged to use this everywhere.